release.config.js

```diff
@@ -3,12 +3,14 @@ module.exports = {
   branches: [
     'master',
     {
-      name: 'beta',
-      prerelease: true,
+      name: 'develop',
+      channel: 'beta',
+      prerelease: 'beta',
     },
     {
-      name: 'alpha',
-      prerelease: true,
+      name: 'next',
+      channel: 'next',
+      prerelease: 'next',
```

Comment:
I'm not sure we keep the next as prerelease or not?

Reply:
I think it should be prerelease. Shouldn't it be a boolean instead?